update-motd.d/99-esm

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+ESM_ENABLED="This Ubuntu 12.04 ESM system is actively receiving extended security updates
```

Comment:
Does it change identity just because ESM is enabled? It's still called Ubuntu 12.04 LTS last time I checked?